web/styles/zulip.css

```diff
@@ -327,7 +327,28 @@ body {
     --color-gear-menu-lighter-text: hsl(0deg 0% 40%);
     --color-gear-menu-blue-text: hsl(217deg 100% 50%);
     --color-header-button-hover: hsl(0deg 0% 0% / 5%);
+    /* This is a technique for directing CSS to do
+       the color mixing ordinarily handled by the
+       alpha channel in hsl(); here, the alpha value
+       is omitted from the hsl() syntax and instead
+       used as the percentage for mixing over top of
+       the navbar background. This is needed so that
+       elements like the unread-count dot can make
+       use of the color, but neither compound alpha
+       values or apply alpha values over different
+       colors, such as the background color on the
+       unread dot. */
+    --color-header-button-hover-no-alpha: color-mix(
+        in srgb,
+        hsl(0deg 0% 0%) 5%,
+        var(--color-background-navbar)
+    );
```

Comment:
We need to copy-paste the value rather than referring to a `var()` if we want postcss-preset-env to compile this down, which we do, because `color-mix()` is [pretty new](https://caniuse.com/mdn-css_types_color_color-mix).

Reply:
We can do that with a comment referencing the variable, so it's easy to find via grep if changing the variable.

Reply:
Updated. Thanks!